examples/Migration/CampaignManagement/CreateCompleteCampaignBothApisPhase1.php

```diff
@@ -140,8 +138,7 @@ private static function createCampaignBudget(
         // Creates a campaign budget operation.
         $campaignBudgetOperation = new CampaignBudgetOperation();
         $campaignBudgetOperation->setCreate($campaignBudget);
-
-        /** @var MutateCampaignBudgetResponse $campaignBudgetResponse */
+        
```

Comment:
Instead of removing this, we should import MutateCampaignBudgetResponse

Reply:
The @var is required for making IDE understand the result from $campaignBudgetServiceClient->mutateCampaignBudgets().
Please still keep that.

Reply:
Right, restored this, added the use clause and also added @var comments and use clauses for all other mutate operations to make it uniform.